packages/sdks/src/components/blocks/blocks-wrapper.lite.tsx

```diff
@@ -30,13 +30,19 @@ export type BlocksWrapperProps = {
   BlocksWrapperProps: any;
 
   children?: any;
+
+  classNameProp?: string;
 };
 
 export default function BlocksWrapper(props: BlocksWrapperProps) {
   const blocksWrapperRef = useRef<HTMLDivElement>();
   const state = useStore({
     get className() {
-      return 'builder-blocks' + (!props.blocks?.length ? ' no-blocks' : '');
+      return (
+        'builder-blocks' +
+        (!props.blocks?.length ? ' no-blocks' : '') +
+        (props.classNameProp ? ' ' + props.classNameProp : '')
+      );
```

Comment:
it'll be easier to use an array so you don't have to remember the `' '` spaces each time:

```suggestion
      return [
        'builder-blocks',
        !props.blocks?.length ? 'no-blocks' : '',
        props.classNameProp,
      ]
        .filter(Boolean)
        .join(' ');
```